src/EnergyPlus/HeatBalanceAirManager.cc

```diff
@@ -3862,7 +3840,7 @@ void GetSimpleAirModelInputs(EnergyPlusData &state, bool &ErrorsFound) // IF err
             } else if (Zone1Num < Zone2Num) { // zone 1 will come first in soln loop, id zone 2 as mate zone
                 ZoneNumA = Zone1Num;
                 ZoneNumB = Zone2Num;
-            } else if (Zone2Num < Zone1Num) { // zone 2 will come first in soln loop, id zone 1 as mate zone
+            } else { // zone 2 will come first in soln loop, id zone 1 as mate zone
```

Comment:
[src/EnergyPlus/HeatBalanceAirManager.cc:3865]:(style),[knownConditionTrueFalse],Condition 'Zone2Num<Zone1Num' is always true